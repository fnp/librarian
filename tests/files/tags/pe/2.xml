<akap>
  Bieży<pe>
  <akap><slowo_obce>bieży</slowo_obce> --- biegnie.</akap>
  <akap>Drugi akapit przypisu.</akap>
  </pe>.
</akap>
