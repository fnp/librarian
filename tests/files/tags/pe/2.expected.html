<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Bieży<a href="#footnote-id1" class="annotation-anchor" id="anchor-id1">[1]</a>.</p>
<div id="footnotes">
<h3>Przypisy</h3>
<div class="fn-pe">
<a name="footnote-id1"></a><a href="#anchor-id1" class="annotation">[1]</a><p>
  <p class="wl paragraph"><em class="foreign-word">bieży</em> — biegnie.</p>
  <p class="wl paragraph">Drugi akapit przypisu.</p>
   [przypis edytorski]</p>
</div>
</div>
</div>
